eng/build.ps1

```diff
@@ -151,11 +151,23 @@ function Set-OptProfVariables() {
 function Check-EditedFiles() {
   # Log VSTS errors for changed lines
   git --no-pager diff HEAD --unified=0 --no-color --exit-code | ForEach-Object { "##vso[task.logissue type=error] $_" }
-  if($LASTEXITCODE -ne 0) {
+  if ($LASTEXITCODE -ne 0) {
     throw "##vso[task.logissue type=error] After building, there are changed files.  Please build locally and include these changes in your pull request."
   }
 }
 
+function Check-RequiredVersionBumps() {
+  # Log VSTS errors for missing required version bumps
+  $versionLineChanged = $false
+  git --no-pager diff --unified --no-color --exit-code -w origin/$env:SYSTEM_PULLREQUEST_TARGETBRANCH HEAD src\Framework\BuildEngineInterface.cs `
+    | Select-String -Pattern "int Version =" | ForEach-Object -process { $versionLineChanged = $true }
+  if (($LASTEXITCODE -ne 0) -and (-not $versionLineChanged)) {
+    throw "##vso[task.logissue type=error] Detected changes in Framework\BuildEngineInterface.cs without a version bump.  " +
```

Comment:
```suggestion
    throw "##vso[task.logissue type=error;sourcepath=src/Framework/BuildEngineInterface.cs] Detected changes in Framework\BuildEngineInterface.cs without a version bump.  " +
```

Not sure if this helps the GitHub annotations without a line number but it shouldn't hurt.

Reply:
Applied, thank you.